### PR TITLE
Refactor daint performance & profile scripts

### PR DESCRIPTION
**Pull request by ofuhrer**
## Purpose

This PRs has changed in purpose. Data now seems to live on /scratch. So changes have been adapted accordingly and updated to latest master.

Add simple generation of profiling output in *.txt and *.html format. See https://jenkins.ginko.ch/view/Owners/job/fv3core-performance-profile/backend=gtx86,experiment=c128_6ranks_baroclinic,slave=daint_submit/ for an example.

### CODE REVIEW

**Review by rheacangeo — APPROVED** [review]

File: .jenkins/actions/run_performance.sh [hunk 1]
```diff
@@ -3,20 +3,12 @@ set -e -x
...
-if [ "$experiment" = "c12_6ranks_standard" ]; then
-    data_path="${TESTDATA_PATH}7.0.0/c12_6ranks_standard"
-fi
-if [ "$experiment" = "c96_6ranks_baroclinic" ]; then
-    data_path="${TESTDATA_PATH}7.2.3/c96_6ranks_baroclinic"
-fi
-if [ "$experiment" = "c128_6ranks_baroclinic" ]; then
-    data_path="${TESTDATA_PATH}7.2.3/c128_6ranks_baroclinic"
-fi
+data_path=${TESTDATA_PATH}/${FORTRAN_SERIALIZED_DATA_VERSION}/${experiment}
```
Comment (twicki) [inline .jenkins/actions/run_performance.sh:11]: can we also apply this change in 
https://github.com/VulcanClimateModeling/fv3core/blob/c6b2081fa600bc4ddc80d5fb144386bebfecb71e/.jenkins/actions/full_cache_build.sh#L11-L19

**Review by twicki — CHANGES_REQUESTED** [review]

**Review by twicki — CHANGES_REQUESTED** [review]
one more place where this is needed

File: .jenkins/actions/run_performance.sh (hunk 1 above)
Comment (ofuhrer) [reply .jenkins/actions/run_performance.sh:11]: Thanks, @twicki, done!

**Review by twicki — APPROVED** [review]
lgtm, thanks

**Comment by ofuhrer** [comment]
launch jenkins

**Pull request merged**